app/views/spree/shared/_payment.html.haml

```diff
@@ -18,5 +18,5 @@
   %p.callout{style: "margin-top: 40px"}
     %strong
       = t :email_payment_description
-  %p{style: "margin: 5px"}= payment_method(@order.payments.last).description
+  %p{style: "margin: 5px"}= payment_method(@order.payments.first).description
```

Comment:
Ok, actually I don't know if I should use the last or the first... 
I had two different text.

Reply:
Ok, I've asked in the issue: https://github.com/openfoodfoundation/openfoodnetwork/issues/6359#issuecomment-832493160

Reply:
Check 2d216de64 
_Use the first completed payment to display payment description_

Reply:
Finally, use the first payment with `completed` or `checkout`state:  5e0180ef3c9212877f7a7cd989bf7f3192e42eca